index.html

```diff
@@ -0,0 +1,201 @@
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li class="category-list"><a href="index.html">html</a></li>
+                        <li class="category-list"><a href="index.html">css</a></li>
+                        <li class="category-list"><a href="index.html">javaScript</a></li>
+                    </ul>
+                    <button type="button" class="project" data-modal-target="#modal0">See Project</button>
+                </section>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-photo" id="photo3" data-modal-target="#modal1">
+                </div>
+                <section class="grid-item-section">
+                    <h2 class="title">Tonic</h2>
+                    <ul class="sub-title">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li class="category-list"><a href="index.html">html</a></li>
+                        <li class="category-list"><a href="index.html">css</a></li>
+                        <li class="category-list"><a href="index.html">javaScript</a></li>
+                    </ul>
+                    <button type="button" class="project" data-modal-target="#modal1">See Project</button>
+                </section>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-photo" id="photo2" data-modal-target="#modal0">
+                </div>
+                <section class="grid-item-section">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <ul class="sub-title">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                    <ul class="category">
+                        <li class="category-list"><a href="index.html">html</a></li>
+                        <li class="category-list"><a href="index.html">css</a></li>
+                        <li class="category-list"><a href="index.html">javaScript</a></li>
+                    </ul>
+                    <button type="button" class="project" data-modal-target="#modal0">See Project</button>
+                </section>
+            </div>
+        </div>
+        
+        <div class="modal0" id="modal0">           
+        </div>
+        <div class="modal1" id="modal1">
+        </div>
+        <div class="modal2" id="modal2">
```

Comment:
The modal is not responsive to other viewpoints. Kindly right-click on your page and click on `inspect` to see the difference in other viewpoints.

![image](https://user-images.githubusercontent.com/38088818/175529447-a9d6e7cf-a3ff-4f07-bd51-fb3d661a971e.png)

Also in mobile view, the spacing of your modal is uneven:

![image](https://user-images.githubusercontent.com/38088818/175529967-6791af35-45ac-48d8-b7ab-407fbc4c39a1.png)